modules/ROOT/pages/as2/AS2-connector-examples-6-0.adoc

```diff
@@ -0,0 +1,90 @@
+|Requester config - MDN MIC Signature Algorithm| The algorithm with which MIC is calculated for the MDN signature. Request partner to respond with a signed MDN using this algorithm for calculating MIC. The message is `UNSIGNED` by default.
+|Requester config - Request a receipt| Option to request a receipt from the partner. `SIGNED_OPTIONAL` by default.
+|Requester config - Request delivery URL| Asynchronous receipt delivery URL that points to the MDN listener endpoint.
+|Requester config - Encryption Algorithm| The encryption algorithm for encrypting the message. `UNENCRYPTED` by default.
+|Requester config - MIC Signature Algorithm| The algorithm with which MIC is calculated for the message signature.  The message is `UNSIGNED` by default.
+|Requester config - Content transfer encoding| The content transfer encoding. `BINARY` by default.
+|Requester config - Compression type| ZLIB is the only supported compression and decompression type. `NONE` by default.
+|Keystore config - Keystore Path| The path of the keystore where certificates and keys are provided.
+|Keystore config - Keystore Password| The password for the keystore.
+|Keystore config - Private key Password| The private key password, if required.
+|===
++
+. Configure a connection to the partner AS2 endpoint by providing the `Partner Base URL`.
+. If you want to configure other connection details (such as TLS, proxy, basic authentication, and timeouts) do so now.
+. Send the file or message by either using `Send with sync mdn` or `Send with async mdn`:
+
+** The payload is the message content to be sent.
+
+** Filename is the file name of the message being sent.
+
+** The Boolean required to configure if the process should fail or if the receipt status is not processed.
+
+. If you choose to request an asynchronous receipt, you must configure the `MDN Listener`.
+
+=== Configure an MDN Listener
+
+Place the connector’s `MDN Listener` as a source in your flow as applicable and reference the global config defined here.
+
+[%header%autowidth.spread]
+|===
+|Key| Description
+|Self config - AS2 OID| Sender partner’s OID.
+|Self config - x509 alias| Sender’s certificate and/or private key alias in the keystore.
+|Self config - Email| Sender partner’s email.
+|Partner config - AS2 ID| Receiver partner’s OID.
+|Partner config - x509 alias| Receiver’s Certificate and/or private key alias in the keystore.
+|Partner config - Email| Receiver partner’s email.
+|HTTP Listener config| Reference to HTTP Listener global config.
+|Keystore config - Keystore Path| The path of the keystore where certificates and keys are provided.
+|Keystore config - Keystore Password| The password for the keystore.
+|Keystore config - Private key Password| The private key password if required.
+|===
+
+=== Example: Configure an AS2 Listener for Receiving a Message
+
+. Place the connector’s `AS2 Listener` as a source in your flow as applicable for your needs and reference the global config as defined here.
```

Comment:
"Place as a source in your flow" just sounds odd to me. 
Do we want to emphasize the location or the action of "placing" or do we want to emphasize it's designation as a source by "specifying" or "establishing"?

Reply:
Also, can you "place" a syntax value?
Please review this topic set for how ` ` is used as a format. 
Syntax is a value of something, not the thing itself, so it needs an article and an object: the `AS2 Listener` value or simply `AS2 Listener`.
Otherwise, is it just a thing, not formatted: the AS2 Listener
GLOBAL

Reply:
GLOBALLY spell out "configuration" as a noun and a verb. 
Use "config" only as a syntax value ` ` or label * *.

Reply:
```suggestion
. Insert the connector `AS2 Listener` component as a source in your flow and reference the global element configuration as defined here:
```